crates/cast/src/lib.rs

```diff
@@ -1274,14 +1283,14 @@ impl SimpleCast {
     /// ```
     /// use cast::SimpleCast as Cast;
     ///
-    /// assert_eq!(Cast::to_wei("1", "")?, "1000000000000000000");
```

Comment:
before: anything except exactly the string "gwei" would be parsed as an ether unit...